content/momentum/4/console-commands/4-mailq.md

```diff
@@ -28,4 +28,8 @@ default/#mmove                                   gmail.com       0       0
 
 The `Group/Binding` column shows the group and the binding that a domain belongs to. The `default/#move` entry under the `Group/Binding` column is the cluster message movement binding in the `default` binding. This is a virtual binding for moving messages between nodes. `AQ` shows the items in the active queue and `DQ` shows the items in the delayed queue.
 
-Use this command with the `--domain` *`domain_name`* argument to show the statistics for a specific domain. Any number of domains may be specified. The `--binding` option is used in the same way. Limit the output by using the --limit *`number`* option. The default value for `--limit` is `20`.
\ No newline at end of file
+Use this command with the `--domain` *`domain_name`* argument to show the statistics for a specific domain. Any number of domains may be specified. The `--binding` option is used in the same way. Limit the output by using the --limit *`number`* option. The default value for `--limit` is `20`.
+
+## Attention
+
+This command is unsafe to be executed in production and should be avoided. It is recommended to use the `mailq` HTTP endpoint instead.
```

Comment:
The command is okay to run occasionally.  So I think we should reword it to be less scary.  Something like:

On a busy system, if this command is run frequently it may result in a Momentum watchdog restart.  If mailq data is needed multiple times per hour then it is recommended to use the `mailq` HTTP endpoint instead.

Also, can we link to the new mailq HTTP doc?